docs/content/users/providers/index.md

```diff
@@ -2,6 +2,8 @@
 
 DDEV offers hosting provider integration and sample integrations for [Pantheon](https://pantheon.io), [Platform.sh](https://platform.sh) and [Acquia](https://www.acquia.com) hosting, along with other examples.
 
+Hosting provider integration (`ddev pull <provider>` and `ddev push <provider>`) allows connecting with upstream hosting to get database and user-generated (volatile) files. It does *not* deploy or pull your code. Your code should be under git control, and you pull and push it using your team's strategy. `ddev pull` pulls the **database** and the **user-generated files** from an upstream provider.
```

Comment:
```suggestion
Hosting provider integration allows connecting with your upstream hosting. `ddev pull <provider>` downloads and `ddev push <provider>` uploads the **database** and the **user-generated files** to an upstream provider. It does *not* push (deploy) or pull your code. Your code should be under version control in for example Git. 
```

I am not sure I've tried to make that part more concise. For one i thought it might make sense to avoid the ddev push and pull in parenthesis. i'Ve moved that to the second sentence. i've also striked the last sentence cuz with that in place you meander between the database user generate bit and the code bit back and forth. i was also not sure if the "using your teams strategy" part is really necessary? not everyone is part of a team. and implied anyway if it is the case? the only detail i am unsure about my suggestion is the end of the second sentence, `...to an upstream provider.`. no sure the reference "to" is correct here. for the ddev push it is but for ddev pull technically it would be from?

Reply:
and i've wrote more in general `under version control in for example Git` since it is not sure if Git is the tool of choice even though it is the quasi standard and you also wrote about the quasi standard in docs/content/users/topics/index.md which comes before this part in the TOC. but it is not sure that a visitor read the other page before before therefore i thought going with a more general formulation might be a good idea?

Reply:
Very nice improvement here!